src/docs/pages/component-docs/components/toolbar-menu/examples/within-toolbar.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+export const WITHIN_TOOLBAR = `<mat-toolbar color="primary" style="width: 300px">
+    <button mat-icon-button style="margin-right: 16px">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div>
+        <div class="mat-title" style="line-height: 1rem">Title</div>
```

Comment:
Does not appear centered... 

![image](https://user-images.githubusercontent.com/8997218/192373246-47d72dfe-f1f0-4be7-8b76-8a82e19dbf9f.png)

Reply:
I've slapped a margin on this one to center the content.